KIPs/kip-draft_kgp6.md

```diff
@@ -0,0 +1,734 @@
+---
+kip: 103
+title: Treasury Fund Rebalancing
+author: Aidan (@aidan-kwon), Toniya (@toniya-klaytn)
+discussions-to: https://govforum.klaytn.foundation/t/kgp-6-proposal-to-establish-a-sustainable-and-verifiable-klay-token-economy/157
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-02-24
+---
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the KIP.-->
+
+This proposal suggests a smart contract interface standard that records the rebalance of treasury funds. The main objective is to facilitate the approval and redistribution of treasury funds to new addresses while keeping record of the Retired and Newbie details.
```

Comment:
The explanation seems unclear.
What do Retired and Newbie stand for?
Does 'retired' represent 'previous fund account' such as KGF or KIR, and 'Newbie' represents 'rebalanced fund' such as KCF or KFF?
Or how about do not write 'Retired' and 'Newbie' at simple summary?

Reply:
yes, 'retired' represent 'previous fund account' such as KGF or KIR, and 'Newbie' represents 'rebalanced fund' such as KCF or KFF. As suggested, will to keep it simple will refactor to 
```
The main objective is to facilitate the approval and redistribution of treasury funds to new addresses while keeping record of the previous fund addresses.
```